pkg/chart/bean/bean.go

```diff
@@ -59,9 +60,13 @@ type ChartUpgradeRequest struct {
 }
 
 type ChartRefChangeRequest struct {
-	AppId            int `json:"appId" validate:"required"`
-	EnvId            int `json:"envId" validate:"required"`
-	TargetChartRefId int `json:"targetChartRefId" validate:"required"`
+	AppId               int                          `json:"appId" validate:"required"`
+	EnvId               int                          `json:"envId" validate:"required"`
+	TargetChartRefId    int                          `json:"targetChartRefId" validate:"required"`
+	EnvConfigProperties *bean2.EnvironmentProperties `json:"-"`
+	EnvMetrics          bool                         `json:"-"`
+	UserId              int32                        `json:"-"`
+	Token               string                       `json:"-"`
```

Comment:
why are we sending token, not recommended. Also for every logging of this obj token will get logged...remove this